app/Console/Commands/SeederBackupData.php

```diff
@@ -38,14 +38,23 @@ public function handle()
         $options = $this->options();
 
         // option to use environment to restore or backup to different environment files
-        if (isset($options['env']) && in_array($options['env'], config('backup.filament_seeders.environments')) === true) {
```

Comment:
This is to resolve the passing of a separate seeders environment on to the seeders to make sure it resolves the filename correctly.